segments/helpers.py

```diff
@@ -96,8 +112,8 @@ def refresh_segment(self, segment_id, sql):
             self.remove_segment_membership(segment_id, user_id)
 
         # Copy the new adds and deletes to the member changed list
-        self.redis.sunionstore(new_key, self.segment_member_refresh_key, self.segment_member_refresh_key)
-        self.redis.sunionstore(del_key, self.segment_member_refresh_key, self.segment_member_refresh_key)
+        self.redis.sunionstore(self.segment_member_refresh_key, del_key)
```

Comment:
Does it accept 2 params now ?

Reply:
I took it out entirely -- it's handled in the individual methods now.